app/models/comment.rb

```diff
@@ -5,4 +5,11 @@ class Comment < ApplicationRecord
   belongs_to :creator, class_name: "User"
 
   after_create_commit { broadcast_append_to dom_id(self.ticket, :comments) }
+
+  after_create :send_notifications
+
+  def send_notifications
+    TicketsMailer.commented(self.ticket, self.ticket.creator).deliver
+    TicketsMailer.commented(self.ticket, self.ticket.assignee).deliver if self.ticket.assignee.present?
```

Comment:
We should add the `creator != assignee` check here, too

Reply:
Yes, sir!